.travis.yml

```diff
@@ -2,21 +2,34 @@ language: python
 
 sudo: false
 
-env:
-  - TOXENV=py27
-  - TOXENV=py34
-  - TOXENV=py35
-  - TOXENV=py36
-  - TOXENV=flake8
+dist: trusty
+
+python:
+  - '3.6'
+  - '3.5'
+  - '3.4'
+  - '2.7'
 
 os:
   - linux
   - osx
 
 matrix:
+  include:
+    - env: TOXENV=py27
```

Comment:
Maybe we can try setting `TOXENV=py` globally? That could make it use only the default python (avoiding the duplicate tests we were getting with the previous versions). Not sure, but worth a try :wink: